main.go

```diff
@@ -175,11 +180,38 @@ func handle(w http.ResponseWriter, r *http.Request) {
 		defer f.Close()
 		http.ServeContent(w, r, "main.wasm", time.Now(), f)
 		return
+
+	case "__wait":
+		waitForUpdate(w, r)
+		return
+	case "__notify":
+		notifyWaiters(w, r)
+		return
```

Comment:
I prefer `_wait` / `_notify` to `__wait` /`__notify`, though the difference is slight.